src/Cli/ConfigGenerator.cs

```diff
@@ -119,7 +119,12 @@ public static bool TryCreateRuntimeConfig(InitOptions options, FileSystemRuntime
                     // rest is not supported for cosmosdb_nosql yet.
                     if (!RestRuntimeOptions.DEFAULT_PATH.Equals(restPath))
                     {
-                        _logger.LogWarning("Configuration option --rest.path is not honored for cosmosdb_nosql since it does not support REST yet.");
+                        _logger.LogWarning("Configuration option --rest.path is not honored for cosmosdb_nosql since CosmosDB does not support REST.");
+                    }
+
+                    if (options.RestRequestBodyStrict is CliBool.False)
+                    {
+                        _logger.LogWarning("Configuration option --rest.request-body-strict is not honored for cosmosdb_nosql since CosmosDB does not support REST.");
```

Comment:
Need a test to validate whether CLI knows to not add this fields to the config for CosmosDB databaseType. 
Also this would probably get annoying to see in the logs when using cosmos even if the user doesn't define these fields. I'm assuming these warnings are emitted all the time because we set default values for `--rest.path` and `--rest.request-body-strict` no matter the db type?

Reply:
If we log this every time and user can't do anything about it, I would suggest this be categorized as an "Information" log level message.

Reply:
No, these warnings are only emitted when the values are not equal to the default values and the option is included in the init command.

Reply:
Oh, so if it is excluded from the init command, it does not log? Then keeping as warning should be fine. (I would suggest an error , but that would be a breaking change. so can't do that).

Reply:
No, it does not.